app/config.py

```diff
@@ -14,7 +14,7 @@ class SystemConfig(BaseSettings):
 
     model_config = SettingsConfigDict(
         env_prefix=SERVICE_NAME + "SYSTEM_",
-        env_file=ROOT_DIR / Path(".env"),
+        env_file=ROOT_DIR / Path("../.env"),
         env_file_encoding="utf-8",
         extra="ignore",
```

Comment:
This change is a bit wrong. You should update the ROOT_DIR constant instead of changing this in PydanticSettings

Reply:
Also don't forget to test that env variables is setting okay after you change the ROOT_DIR

Reply:
so add to ROOT_DIR = "../"? don`t remember if why we changed it, was it changed automatically ? or we did it ?

Reply:
this is how it should look like: 
ROOT_DIR = Path(__file__).parent/Path("../")
?